### DISCUSSION TITLE
New payment method about SUPPORTER-CERTIFICATE

### DISCUSSION THREAD

**Discussion by cheese2020** [sandboxie-plus/Sandboxie#1813] [Various ideas and suggestions]
### Is your feature request related to a problem or use case?

In China ,we could not use paypal.but I am really love function such like privacy enhanced mode. So could you use a new payment method for China user

### Describe the solution you'd like

use a new payment method for China user，for example:Alipay,Wechat

**Comment by DavidXanatos** [comment]
how about bitcoin?

**Reply by 0x4E69676874466F78** [reply]
How can I donate via bitcoin to you?

**Reply by isaak654** [reply]
See [CONTRIBUTING.md](https://github.com/sandboxie-plus/Sandboxie/blob/master/CONTRIBUTING.md).

**Comment by cheese2020** [comment]
OKOK, I think we can use USDT,but to tell you the truth,i am a senior high school student,it maybe not a lot of money

**Comment by eebssk1** [comment]
Bitcoin platforms in China are already sunsetting in response to governance.
To use it you still need to use offshore payment methods and cross the wall, which is not relevant for this case.

**Comment by isaak654** [comment]
I'm going to add the original suggestion of @mitchcapper for future reference:

> I have noticed a few times now people inquiring about paying by bitcoin.  If you think there is some decent demand beyond a few users for that you could look at https://bitpay.com/business  it has a 1% transaction fee (but that beats paypal) and they have the option for instant fiat conversion that guarantees you the specific fiat amount you set and adjusts the bitcoin on the fly for people purchasing (it locks the amount for 15 minutes or something during checkout so the rate doesn't change on them).  They also take a few other forms of crypto other than BTC.  A few days after payment they can do a sweep into your bank acocunt.    They support payment buttons like paypal: https://bitpay.com/docs/ecommerce#buttons .  They are not a great company, but they are one of the few that can completely remove that hassle. I have had to work with them once before for another project.
